pkg/controller/controller.go

```diff
@@ -730,7 +731,7 @@ func (p *csiProvisioner) Provision(ctx context.Context, options controller.Provi
 	createCtx := markAsMigrated(ctx, result.migratedVolume)
 	createCtx, cancel := context.WithTimeout(createCtx, p.timeout)
 	defer cancel()
-	klog.V(5).Infof("CreateVolumeRequest %+v", req)
+	klog.V(5).Infof("CreateVolumeRequest %+v", protosanitizer.StripSecrets(req))
```

Comment:
I think we could even remove these logs entirely. It should already be logged properly via https://github.com/kubernetes-csi/csi-lib-utils/blob/44b91e2a7b0af8199fc65c1142f664fd5f7abb96/connection/connection.go#L112

@chrishenzie can you followup? I think we'll also want to backport the fixes

Reply:
Fix here:
https://github.com/kubernetes-csi/external-provisioner/pull/604